SPECS/ncompress/ncompress.spec

```diff
@@ -0,0 +1,51 @@
+Summary:        A fast, simple LZW file compressor
+Name:           ncompress
+Version:        5.0
+Release:        1%{?dist}
+License:        Unlicense
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/vapier/ncompress
+#Source0:       https://github.com/vapier/%{name}/archive/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+BuildRequires:  gcc
+BuildRequires:  glibc-devel
+BuildRequires:  make
+
+%description
+
+This is (N)compress. It is an improved version of compress 4.1.
+
+Compress is a fast, simple LZW file compressor. Compress does not have the highest compression rate,
+but it is one of the fastest programs to compress data. Compress is the defacto standard in the UNIX
+community for compressing files. (N)compress 4.2 introduced a special, fast compression hash algorithm.
+This algorithm uses more memory than the old hash table. If you don't want the faster hash table algorithm
+et 'Memory free for compress' below 800000. Starting with compress 3.0, the output format changed in a
+backwards incompatible way. This is not a big deal as compress 3.0 was first released in Jan 1985, while
+the first release of compress was available less than a year prior. There shouldn't be any need to produce
+files that only older versions of compress would accept. Newer versions of compress are still able to 
+handle the output of older versions though -- i.e. compress 3.0+ is able to decompress files produced
+by compress 2.0 and older.
+
+%prep
+%autosetup
+
+%build
+make %{?_smp_mflags}
+
+%install
+make PREFIX=%{_prefix} DESTDIR=%{buildroot} install_core
+
+# %%check
```

Comment:
I think we can remove this comment then.

Reply:
I think I disagree here- we currently don't have another way to distinguish between packages that lack %check sections because they don't have tests, and packages that lack %check sections because no one has bothered to write one. 

I think this serves a dual purpose as (a) a reminder to check for new package tests if this package is upgraded, and (b) a way to save investigation time for anyone assigned to some sort of blanket "enable more package tests" task in the future.

Reply:
I see your point, makes sense.